### DISCUSSION TITLE
Getting xbee to run zigbee2mqtt in docker??

### DISCUSSION THREAD

**Discussion by kakemanx** [Koenkk/zigbee2mqtt#9990] [Devices]
I have trialed AND failed for hours and hours now, and just can't get it to work :/*
If anyone have any pointers i would greatly appreciate it :)

Some info, running NUC with an Aeotech zwave dongle and a Conbee ii (zha) and want to run the xbee with zigbee2mqtt for a few devices..

Mqtt running in docker

I use this docker command:
sudo docker run \
   --device=/dev/ttyUSB0 \
   -v $(pwd)/data:/app/data \
   -v /run/udev:/run/udev:ro \
   -e TZ=Europe/Amsterdam \
   koenkk/zigbee2mqtt
   
   And this is my configuration.yaml:
     GNU nano 4.8                                  /home/kakemanx/data/configuration.yaml                                             
# Home Assistant integration (MQTT discovery)
homeassistant: false

# allow new devices to join
permit_join: true

# MQTT settings
mqtt:
  # MQTT base topic for zigbee2mqtt MQTT messages
  base_topic: zigbee2mqtt
  # MQTT server URL
  server: 'mqtt://192.168.0.150' (have also tried localhost)
  # MQTT server authentication, uncomment if required:
  # user: my_user
  # password: my_password

# Serial settings
serial:
  # Location of CC2531 USB sniffer
  port: /dev/ttyUSB0
baudrate: 57600 (this was a test, because i didnt know what baudrate is default.. xbee iis @57600)

I get these errors:

kakemanx@haserver:~$ sudo docker run    --device=/dev/ttyUSB0    -v $(pwd)/data:/app/data    -v /run/udev:/run/udev:ro    -e TZ=Europe/Amsterdam    koenkk/zigbee2mqtt
Using '/app/data' as data directory
Zigbee2MQTT:info  2021-12-04 01:21:19: Logging to console and directory: '/app/data/log/2021-12-04.01-21-18' filename: log.txt
Zigbee2MQTT:info  2021-12-04 01:21:19: Starting Zigbee2MQTT version 1.22.0 (commit #0a1fccf
)
Zigbee2MQTT:info  2021-12-04 01:21:19: Starting zigbee-herdsman (0.13.164)
Zigbee2MQTT:error 2021-12-04 01:21:40: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2021-12-04 01:21:40: Failed to start zigbee
Zigbee2MQTT:error 2021-12-04 01:21:40: Check https://www.zigbee2mqtt.io/information/FAQ.html#help-zigbee2mqtt-fails-to-start for possible solutions
Zigbee2MQTT:error 2021-12-04 01:21:40: Exiting...
Zigbee2MQTT:error 2021-12-04 01:21:40: Error: Failed to connect to the adapter (Error: SRSP - SYS - ping after 6000ms)
    at ZStackAdapter.start (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/zStackAdapter.ts:102:27)
    at Controller.start (/app/node_modules/zigbee-herdsman/src/controller/controller.ts:123:29)
    at Zigbee.start (/app/lib/zigbee.ts:58:27)
    at Controller.start (/app/lib/controller.ts:100:27)
    at start (/app/index.js:97:5)
    
    If anyone could help i would be very greatfull :)

**Comment by francisp2** [comment]
>want to run the xbee with zigbee2mqtt for a few devices..

xbee is not a supported adapter
